baseplate/observers/tracing.py

```diff
@@ -282,10 +292,6 @@ def _serialize(self) -> Dict[str, Any]:
 
         return self._to_span_obj(annotations, self.binary_annotations)
 
-    def record(self) -> None:
```

Comment:
🔕 I'm assuming this got removed from the interface and we just missed this one?

Reply:
nah, just removing a pretty unnecessary function. it was always like this :man_shrugging: